platform/osgi-platform/src/main/java/org/motechproject/server/osgi/status/PlatformStatusManagerImpl.java

```diff
@@ -20,13 +25,34 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PlatformStatusManagerImpl.class);
 
+    private static final String EVENT_SUBJECT = "All bundles started.";
```

Comment:
1) this should be a public constant in PlatformConstants
2) the subject should be named according to convention - org/motechproject/osgi/event/XXX , XXX can be MODULES_STARTED for example